pkgs/applications/science/biology/bftools/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, lib, pkgs, fetchurl, unzip, openjdk }:
+
+stdenv.mkDerivation rec {
+  name = "bftools-${version}";
+  version = "5.9.2";
+
+  src = fetchurl {
+    url = "http://downloads.openmicroscopy.org/bio-formats/${version}/artifacts/bftools.zip";
+    sha256 = "02x8p07z7js41kgiblkr9k8v9v4ka8rm7pg3f1vy9s5p6hmpfvnx";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./* $out/
```

Comment:
Is there are reason to put things at the root of the derivation ? `$out/bin$` should be enough.

Reply:
There are a number of other .sh files, and one .jar file, and the executable expect to be in the same directory. Didn't think that all these files should go in bin? Let me know

Here are the unzipped files:
```
-rw-r--r--  1 tyler tyler 2.1K Jul  3  2018 bf.bat
-rwxr-xr-x  1 tyler tyler  374 Jul  3  2018 bfconvert
-rw-r--r--  1 tyler tyler  298 Jul  3  2018 bfconvert.bat
-rwxr-xr-x  1 tyler tyler 2.1K Jul  3  2018 bf.sh
-rw-r--r--  1 tyler tyler  31M Sep  3 10:51 bioformats_package.jar
-rw-r--r--  1 tyler tyler 1.2K Jul  3  2018 config.bat
-rw-r--r--  1 tyler tyler 1.1K Jul  3  2018 config.sh
-rwxr-xr-x  1 tyler tyler  375 Jul  3  2018 domainlist
-rw-r--r--  1 tyler tyler  299 Jul  3  2018 domainlist.bat
-rwxr-xr-x  1 tyler tyler  379 Jul  3  2018 formatlist
-rw-r--r--  1 tyler tyler  303 Jul  3  2018 formatlist.bat
-rwxr-xr-x  1 tyler tyler 1.5K Jul  3  2018 ijview
-rw-r--r--  1 tyler tyler 1.3K Jul  3  2018 ijview.bat
-rw-r--r--  1 tyler tyler  12K Jan 10 11:59 .ijview.swp
-rw-r--r--  1 tyler tyler  281 Jul  3  2018 logback.xml
-rwxr-xr-x  1 tyler tyler  401 Jul  3  2018 mkfake
-rw-r--r--  1 tyler tyler  319 Jul  3  2018 mkfake.bat
-rwxr-xr-x  1 tyler tyler  428 Jul  3  2018 showinf
-rw-r--r--  1 tyler tyler  358 Jul  3  2018 showinf.bat
-rwxr-xr-x  1 tyler tyler  418 Jul  3  2018 tiffcomment
-rw-r--r--  1 tyler tyler  348 Jul  3  2018 tiffcomment.bat
-rwxr-xr-x  1 tyler tyler  356 Jul  3  2018 xmlindent
-rw-r--r--  1 tyler tyler  280 Jul  3  2018 xmlindent.bat
-rwxr-xr-x  1 tyler tyler  352 Jul  3  2018 xmlvalid
-rw-r--r--  1 tyler tyler  276 Jul  3  2018 xmlvalid.bat
```

Reply:
then they should probably go in $out/libexec or something like that.

Reply:
That will cause the software to break as there are hardcoded assumptions to the relative path

Reply:
I agree with @symphorien . If we wanted to do things according to FHS then bf.sh would go to libexec/ and the jar file would probably go to share/java/. The xml file is probably just garbage.